### PR TITLE
Fix WebCommandError remote woes

### PR DESCRIPTION
**Pull request by JackMc**
For some reason I like the word 'woes'.

Addresses #146

### CODE REVIEW

File: datacats/docker.py [hunk 1]
```diff
@@ -103,14 +103,15 @@ def _get_docker():
 
 class WebCommandError(Exception):
 
-    def __init__(self, command, container_id, logs):
+    def __init__(self, command, container_id, logs, ssh=False):
```
Comment (dborzov) [inline datacats/docker.py]: so you are suggesting using this ssh argument as a flag to differentiate between the remotecommand-specific problems and all others?

Comment (JackMc) [inline datacats/docker.py]: Yes, I couldn't find out a way to differentiate them though

That's what I need feedback on :)

Comment (JackMc) [inline datacats/docker.py]: Is the only place where it generates the SSH key?

Comment (dborzov) [inline datacats/docker.py]: I think that that approach can get really confusing after a while. The argument name is not very telling and will need a docstring/comment at least. Another thing is that it we differentiate between classes of errors with dedicated Exception classes elsewhere and here we differentiate between the two classes with True | False argument values.

I am voting for creating a dedicated type of exception for these issues:
```
class RemoteCommandError(WebCommandError):
      ... 
```
I think it is the pythonic way.

Comment (JackMc) [inline datacats/docker.py]: The problem there is that then we need some way to tell if the current command is remote-related.. 

I agree it's more pythonic though

Comment (dborzov) [inline datacats/docker.py]: Good point. Do you mind if I address this? I will just go through all the occurances manually.

Comment (JackMc) [inline datacats/docker.py]: Sure, go ahead. :) 

I'll close this one

Comment (dborzov) [inline datacats/docker.py]: Danke shon :blush:

**Pull request closed without merging**